### PR TITLE
TER-64 TD dedup for fullload

### PR DESCRIPTION
**Pull request by jirkasemmler**
Jira: TER-64

Before asking for review make sure that:

- [x] In the case of **modified import queries**, I checked that the [CustomQuery component](https://github.com/keboola/app-custom-query-manager) is updated.

---

zatim jen pro FullLoad

### CODE REVIEW

**Comment by martinjunger** [comment]
👀

**Review by martinjunger — APPROVED** [review]
LGTM

File: src/Backend/Teradata/ToFinalTable/SqlBuilder.php [hunk 1]
```diff
@@ -135,12 +139,63 @@ public function getInsertAllIntoTargetTableCommand(
...
+
+        return sprintf(
+            'INSERT INTO %s (%s) %s',
+            $deduplication,
+            $this->getColumnsString($deduplicationTableDefinition->getColumnsNames()),
+            $depudeSql
+        );
+    }
+
     public function getTruncateTableWithDeleteCommand(
```
[the remark this answers is not in the document]
Comment (jirkasemmler) [reply src/Backend/Teradata/ToFinalTable/SqlBuilder.php:192]: ostatni backendy jej maji taky a jmenuje se to `getTruncateTableWithDeleteCommand` takze i kdyz na TD neni jina varianta, tak bych to drzel jednotne

File: tests/functional/Teradata/FullImportTest.php [hunk 2]
```diff
@@ -94,6 +95,127 @@ public function testLoadToFinalTableWithoutDedup(): void
...
+            $stagingTable,
+            $destination,
+            $options,
+            $importState
+        );
+
+        self::assertEquals(6, $destinationRef->getRowsCount());
+    }
+
+
```
[the remark this answers is not in the document]
Comment (jirkasemmler) [reply tests/functional/Teradata/FullImportTest.php:218]: :D to by bylo prakticky i neekologicke kvuli tomu poustet build :D

**Pull request merged**